db/User.js

```diff
@@ -1,106 +1,125 @@
-const conn = require('./conn');
+const conn = require("./conn");
 const { Sequelize } = conn;
+const { STRING, BOOLEAN } = Sequelize;
 
-const jwt = require('jsonwebtoken');
-const bcrypt = require('bcrypt');
+const jwt = require("jsonwebtoken");
+const bcrypt = require("bcrypt");
 
-
-const User = conn.define('user', {
+const User = conn.define("user", {
   username: {
-    type: Sequelize.STRING
+    type: STRING,
+    allowNull: false,
   },
   password: {
-    type: Sequelize.STRING
-  }
+    type: STRING,
+    allowNull: false,
+  },
+  isAdmin: {
+    type: BOOLEAN,
+    defaultValue: false,
+  },
+  email: {
+    type: STRING,
+    validate: {
+      isEmail: true,
+    },
+  },
+  imageUrl: {
+    type: STRING,
+    // validate:{
+    //   isUrl: true,
+    // }
+  },
```

Comment:
Maybe, see if you can make a custom` isUrl` validation with regex. 
https://github.com/sequelize/sequelize/issues/9086

Reply:
Delete this commented code